pkg/golang/build.go

```diff
@@ -116,7 +116,6 @@ func (c Environ) Env() []string {
 		cgo = 1
 	}
 	env = append(env, fmt.Sprintf("CGO_ENABLED=%d", cgo))
-	env = append(env, "GO111MODULE=off")
```

Comment:
Why remove this line?

Reply:
because we're on go1.16, we should go with its setting